src/lib/gossip.ts

```diff
@@ -40,7 +40,7 @@ export class Gossip {
   // If callback evaluates to true, the reason we were fetching
   // the mutex has already been fulfilled.
   claimMutex(retries: number, callback?: () => boolean): boolean {
-    sleep(100);
+    sleep(Math.min(Math.pow(2, retries) + Math.round(Math.random() * 1000), 32 * 1000));
```

Comment:
Just need a bump, not exponential. Say 200.

Reply:
can add a jiggle of ${players.length} too

Reply:
There's still value to be had with going exponential. We could cap it off a lot sooner than 32 seconds. The jiggle can also be reduce from 1s to 100ms?

Reply:
The jiggle is to avoid having everyone everyone contend for the mutex at once.

I really think the exponential backoff is more trouble than its worth. Can we please move it to another method, and cap it at 400ms?

Reply:
Capped off at 400ms, and reduced jiggle to 10ms max.